src/CartLatticeAccess.hpp.Rt

```diff
@@ -253,18 +250,22 @@ resolve.symmetries = function(D) {
 
 ?>
 
+CudaDeviceFunction flag_t getContainerType(const CartLatticeContainer& container, int x, int y, int z) {
```

Comment:
This name is misleading, as it suggest container type. Maybe `getNodeType`? And then we'll overload it for different lattices.